src/components/table/ProfileEditView.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { FC } from 'react';
```

Comment:
czemu to jest w `components/table`? Ten komponent jest jakkolwiek powiazanny z table?

Reply:
przypadek, nic związanego z table